cmake/suppressions/tsan.suppressions

```diff
@@ -0,0 +1,3 @@
+# TSAN suppression for known issues.
+# Possible data race during ittnotify initialization. Low impact.
+race:__itt_nullify_all_pointers
```

Comment:
from this line it is not clear that problem is library initialization. Can we narrow it ?

Reply:
ITT API consists of two parts: a static part and a dynamic part. The dynamic part is specific for a tool and distributed only with a particular tool. The static part is integrated into oneTBB.
Initially all pointers to ITT API are initialized with similar to `__itt_someApiCall_init` function. In case collecting data, dynamic part are loaded and pointer will be initialized with correct function from dynamic part, otherwise it will be initialized with `nullprt` and function call would be ignored.
The read/write to a word-sized in aligned location will be atomic on the most systems, but it's not guaranteed by standard.
```
static void __cdecl __itt_someApiCall_init( params... );
typedef void __cdecl __itt_someApiCall_t( params... );
__itt_someApiCall_t* __itt_someApiCall_ptr_ = __itt_someApiCall_init;
static void __cdecl __itt_someApiCall_init( params... ) {
    if (!__itt__ittapi_global.api_initialized && __itt__ittapi_global.thread_list == 0) {
        __itt_init_ittlib_name(...);
    }
    if (__itt_someApiCall_ptr_ && __itt_someApiCall_ptr_ != __itt_someApiCall_init) {
        __itt_someApiCall_ptr_( params... );
    }
}
```
[TSAN Logs](https://github.com/oneapi-src/oneTBB/files/7553321/tsan_logs.txt)

Reply:
@vlserov thanks for the clarification